static/js/timerender.js

```diff
@@ -64,7 +64,7 @@ exports.render_now = function (time, today = new Date()) {
 exports.last_seen_status_from_date = function (last_active_date, current_date = new Date()) {
     const minutes = differenceInMinutes(current_date, last_active_date);
     if (minutes <= 2) {
-        return i18n.t("Just now");
+        return i18n.t("Just now", {interpolation: {escapeValue: false}});
```

Comment:
There’s no interpolation in this call, so I would not expect this to change anything.